.travis.yml

```diff
@@ -9,6 +9,13 @@ python:
   - "pypy"
   - "pypy3"
 
+# work-around for Python 3.7 which is on a non-standard Travis Ubuntu release (Xenial)
+matrix:
+  include:
+    - python: 3.7
+      dist: xenial
```

Comment:
I don't mind moving the entire matrix to xenial. That should still support all the python versions.

Reply:
Apparently Travis xenial does not support regular pypy / pypy3; I've kept those on trusty.  Relevant issue is https://travis-ci.community/t/pypy-2-7-on-xenial/889 .